### PR TITLE
Do not track unmodified relations

### PR DESCRIPTION
**Pull request by johnnyshields**

### CODE REVIEW

**Comment by dblock** [comment]
Needs some `rubocop -a ; rubocop --auto-gen-config` @johnnyshields

File: spec/unit/attributes/update_spec.rb [hunk 1]
```diff
@@ -110,6 +110,44 @@ class EmbOne
...
+
+        let(:obj_one) { DummyUpdateModel.new }
+        let(:base) { described_class.new(obj_one) }
+        let(:changes) do
+          { 'dummy_embedded_model' => [{ 'em_foo' => 'Em-Foo', 'em_bar' => 'Em-Bar' }, { 'em_foo' => 'Em-Foo', 'em_bar' => 'Em-Bar' }] }
+        end
+        subject { base.attributes }
+        it { expect(subject.keys).to_not include 'dummy_embedded_model' }
+
+        after(:all) do
```
Comment (dblock) [inline spec/unit/attributes/update_spec.rb]: Instead of doing this you can `Class.new` a model dynamically, maybe easier.

**Comment by johnnyshields** [comment]
@JagdeepSingh please take over on this PR

**Comment by JagdeepSingh** [comment]
Raised replacement PR here: https://github.com/mongoid/mongoid-history/pull/210

**Comment by JagdeepSingh** [comment]
@dblock @johnnyshields This can be closed now.

**Pull request closed without merging**